app.py

```diff
@@ -1,27 +1,63 @@
-from flask import Flask, request
 from database import db
-import user
+from flask import Flask, request
+from models import User, Business, Point
 
 app = Flask(__name__)
 
 app.config.from_object('config.Config')
 db.init_app(app)
 
+
 @app.route("/")
 def hello():
-    return "Hello World!"
+    return "Hello Fidelity!"
+
 
 @app.route("/users", methods=["POST"])
 def create_user():
     email = request.form["email"]
 
-    # work with sess
-    new_user = user.User(email)
+    new_user = User(email)
     db.session.add(new_user)
     db.session.commit()
 
     return "Created User! %s " % email
 
 
+@app.route("/users/<int:id>", methods=["GET"])
+def get_user(id):
+    user = User.query.get(id)
```

Comment:
It's a pretty similar but not exactly the same. SqlAlchemy is a data mapper, we use db.session for that:
To get we use a static method "User.query..." is not an instance, is a class.
To save it:
user_instance = User
user_instance.name = 'Mickey Mouse'
db.session.add(user_instance)
db.session.commit()  <--- Boom, is in db